docs/user/index.rst

```diff
@@ -87,6 +87,7 @@ to help you create fantastic documentation for your project.
    /single_version
    /science
    /commercial/organizations
+   /server-side-search/index
```

Comment:
This page doesn't seem like explanation, but feature reference to me? It doesn't really talk at all about search at a high level, but about our implementation?

Reply:
Currently, it _explains_ features.

Do you think that it will be better to move into the current feature reference section?

https://docs.readthedocs.io/en/diataxis-main/reference/features.html

Reply:
Yes, I think that is where we should be talking about our features explicitly, not about higher level concepts like explanation. I imagine the higher level content would be things like "What to think about when implementing search" and "ways to write content to ensure good searchability". Perhaps I'm thinking a bit too abstractly for our explanation, but I think anything that's explaining our product itself seems like a feature reference.

Reply:
It's a really good definition and distinction. This can also help to avoid the "marketing effect" in documentation.

I'll move it :+1: